src/components/Navbar.jsx

```diff
@@ -18,48 +20,45 @@ const Navbar = () => {
     return (
         <div className='w-full font-rob font-bold       '>
 
-            <nav className='flex text-white min-h-[6rem]  bg-[#000B70] justify-between py-8 px-4 uppercase items-center gap-4 sm:px-8   '>
-
 
+            <nav className='flex text-white min-h-[6rem]  bg-[#000B70] justify-between py-8 px-4 uppercase items-center gap-4 sm:px-8   '>
 
 
+                <Link to="/" className=' hover:scale-[1.1] transition duration-500'> <img src={Logo} alt="" className='w-[12rem]   md:w-[16rem]  lg:w-[20rem]' /> </Link>
 
-                <Link to="/" className=''> <img src={Logo} alt="" className='w-[12rem]   md:w-[16rem]  lg:w-[20rem]' /> </Link>
 
                 <ul className='hidden  lg:flex lg:gap-12 lg:text-[1.2rem] lg:items-center '>
 
-                    <li> <Link to="/lista"> Lista </Link></li>
-                    <li> <Link to="/posilki"> Posilki </Link> </li>
-                    <li> <Link to="/twojelisty"> Twoje Listy </Link>  </li>
+                    <li className='hover:text-green-500 hover:scale-[1.2] transition duration-500'> <Link to="/lista"> Lista </Link></li>
+                    <li className='hover:text-green-500 hover:scale-[1.2] transition duration-500'> <Link to="/posilki"> Posilki </Link> </li>
+                    <li className='hover:text-green-500 hover:scale-[1.2] transition duration-500'> <Link to="/twojelisty"> Twoje Listy </Link>  </li>
```

Comment:
tutaj DRY - można to wyciągnąć do pomocniczego komponentu lub powtarzające się klasy trzymać w zmiennej